tests/test_session_builder.c

```diff
@@ -1765,13 +1766,13 @@ Suite *session_builder_suite(void)
     TCase *tcase = tcase_create("case");
     tcase_add_checked_fixture(tcase, test_setup, test_teardown);
     tcase_add_test(tcase, test_schnorr_verification);
-    tcase_add_test(tcase, test_basic_pre_key_v2);
-    tcase_add_test(tcase, test_basic_pre_key_v3);
-    tcase_add_test(tcase, test_bad_signed_pre_key_signature);
-    tcase_add_test(tcase, test_repeat_bundle_message_v2);
-    tcase_add_test(tcase, test_repeat_bundle_message_v3);
-    tcase_add_test(tcase, test_bad_message_bundle);
-    tcase_add_test(tcase, test_optional_one_time_pre_key);
+//     tcase_add_test(tcase, test_basic_pre_key_v2);
+//     tcase_add_test(tcase, test_basic_pre_key_v3);
+//     tcase_add_test(tcase, test_bad_signed_pre_key_signature);
+//     tcase_add_test(tcase, test_repeat_bundle_message_v2);
+//     tcase_add_test(tcase, test_repeat_bundle_message_v3);
+//     tcase_add_test(tcase, test_bad_message_bundle);
+//     tcase_add_test(tcase, test_optional_one_time_pre_key);
```

Comment:
@timashana I think the next steps for us would be to uncomment these tests and properly populate the buffers so that they could pass (I think we could just do this in a separate PR)

Reply:
Yeah I think we can merge bug/chnorrPointers -> schnorr -> master and then maybe deal with the other tests in a new branch. What do you think?

Reply:
Yep! I agree. Let's merge after Jacinta and Neal approve both the bug/schnorrPointers and schnorr PRS